Jenkinsfile

```diff
@@ -302,23 +309,24 @@ pipeline {
                         git show-ref --verify refs/remotes/origin/$brew_base_branch
                         git push -d origin brew-release-$PACKAGE || true
                         git checkout -b brew-release-$PACKAGE "origin/$brew_base_branch"
-                        ${WORKSPACE}/src/main/scripts/brew-update-to-local
+                        git merge origin/master
```

Comment:
Why is this here? Hasn't it already done a fresh clone of master?

Reply:
If there is not a `brew-staging: ` tag in the PR title, then it will do nothing (because `brew-release-*` branch will already be based on `origin/master`).

If there is a `brew-staging: ` tag, then it will make sure it's up to date. This is important because we don't always know the order that PRs will get merged in (so another K job may go through which updates `origin/master` in the `homebrew-k` repo), so we want to make sure that if the brew-staging branch is set, it merges cleanly into master before applying the later changes.